### ISSUE TITLE
Call for Volunteers to Condense Notes.

### ISSUE THREAD

**Issue by alexfornuto** [writethedocs/attendee-notes#9]
Now that the conference is over for this year, we can begin looking through the submitted notes, and making a best of file for talks with multiple notes. If you want to spearhead the process for a particular talk, please comment on this issue, or make a new issue and/or PR for that section.

As this is the first time we're doing something like this, we're playing it by ear. If you have ideas for a better method of doing this, please let it be known!

**Comment by dsmatlak** [comment]
Hi Alex,

I'm available to help consolidate the notes from the 2016 WTD NA conference. 

Thanks,
David

**Comment by alexfornuto** [comment]
Thanks @dsmatlak I've merged your PR, as well as several from @smoreface, and @ryangyoung has also contributed.

I hope to have time to start making PR's with consolidated notes per talk, taking all the information from each person's notes. I'll leave them as PR's for discussion, to make sure all contributors feel like the summarized notes encapsulate all that they should.

**Comment by ryangyoung** [comment]
Alex, I'd be happy to help out with this, too. It sounds like you're thinking of creating one "Notes" for each talk, which seems reasonable. Were you thinking of leaving each individual's notes in the repo as well? I could see value in retaining both.

**Comment by alexfornuto** [comment]
Yes @ryangyoung, once it looks like we have individual notes from everyone who's going to contribute, we can divide up the talks between what seems to be the three of us (or more, if we get more volunteers), to make a meta-notes file per talk. Each individual's notes will also remain in the folders. 

I wish there was a way that GitHub could track which lines came from which individual's files, but I can't think of any reasonable method to do so. If you, or anyone with a higher GitHubFu belt than I, knows of a way, please share.
